src/bun.js/ipc.zig

```diff
@@ -475,7 +479,7 @@ fn NewSocketIPCHandler(comptime Context: type) type {
             _: ?*anyopaque,
         ) void {
             // Note: uSockets has already freed the underlying socket, so calling Socket.close() can segfault
-            log("onClose\n", .{});
+            log("NewSocketIPCHandler#onClose\n", .{});
```

Comment:
It sounds like you want to add a new `Output.scoped` for this, instead of adding a namespace inside of a namespace.